src/RichText.webmodeler.ts

```diff
@@ -0,0 +1,33 @@
+import { Component, createElement } from "react";
+import { RichText, RichTextProps } from "./components/RichText";
+import TextEditorContainer, { RichTextContainerProps } from "./components/RichTextContainer";
+
+// tslint:disable-next-line class-name
+export class preview extends Component<RichTextContainerProps, {}> {
+    render() {
+        return createElement(RichText, preview.transformProps(this.props));
+    }
+
+    private static transformProps(props: RichTextContainerProps): RichTextProps {
+        const valueAttribute = props.stringAttribute ? props.stringAttribute.split(".")[ 2 ] : "";
+
+        return {
+            className: props.class,
+            editorMode: props.editorMode,
+            hasContext: true,
+            readOnly: true,
```

Comment:
Should be not readonly, overlay of WM will disable it. 
Readonly based on its settings